src/FieldVector.jl

```diff
@@ -8,11 +8,18 @@ while a mutable `FieldVector` will behave similarly to an `MVector`.
 
 For example:
 
-    immutable/type Point3D <: FieldVector{3, Float64}
+    struct Point3D <: FieldVector{3, Float64}
         x::Float64
         y::Float64
         z::Float64
     end
+
+or
+
+    mutable struct Point2D{T} <: FieldVector{2, T}
+        x::T
+        y::T
+    end
```

Comment:
I think, to some extent, this might just encourage users down the wrong path?  Personally I've never found a practical use for a mutable point like this.